docker/docker-compose.yml

```diff
@@ -22,24 +22,24 @@ services:
     platform: linux/amd64
     hostname: nbxplorer
     ports:
-    - "32838:32838"
```

Comment:
Pls dont modify the docker-compose, Im seeing all PRs changing it, let's sit down if needed but I would remove it from the PR

Reply:
restored